src/Adapter/MSTest.TestAdapter/Execution/TestAssemblyInfo.cs

```diff
@@ -134,7 +139,21 @@ public void RunAssemblyInitialize(TestContext testContext)
                 {
                     try
                     {
-                        AssemblyInitializeMethod.InvokeAsSynchronousTask(null, testContext);
+                        CancellationTokenSource? timeoutTokenSource = null;
```

Comment:
Could you please add a comment in code that says we don't replace the CTS of the `TestContext` because it can be overrided by sub-classes which would break the behavior.